### DISCUSSION TITLE
Expression-language failed in evaluating dates and date-times

### DISCUSSION THREAD

**Discussion by abeer93** [symfony/symfony#45409] [Q&A]
Hello, 
I'm using [expression-language](https://github.com/symfony/expression-language) to evaluate come conditions (rules) over my system
but the package always returns false in case of using date or date-time condition.
 As Example,
 Condition is
 `data.date >= "2022-02-10 04:46"`
 and data object is
 `[date => "2022-02-14 04:46"]`
 I'm expecting the package will return true but it returns false even if I used only date, not date-time.
 Any support?

**Comment by michaoj** [comment]
From what I read in the doc, the expression language does not support dates

See https://symfony.com/doc/current/components/expression_language/syntax.html#supported-literals

**Comment by chyshyya** [comment]
Use
https://symfony.com/doc/current/components/expression_language/syntax.html#calling-methods

**Comment by ghost** [comment]
https://voutzinos.com/blog/using-symfony2-expression-language/
